client/components/resource-details/ResourceInfo.tsx

```diff
@@ -144,29 +137,164 @@ const Reviews = ({ reviews }: ReviewsProps) => {
   return (
```

Comment:
one thing we might support later in this component is sorting/filtering reviews

Reply:
Added default sorting by timestamp. In the future, we can pass a `sortBy` prop to the `Reviews` component and sort `reviews` according to the prop.